src/other-reprs.md

```diff
@@ -112,26 +112,65 @@ assert_eq!(16, size_of::<MyReprOption<&u16>>());
 
 This optimization still applies to fieldless enums with an explicit `repr(u*)`, `repr(i*)`, or `repr(C)`.
 
-## repr(packed)
+## repr(packed(n))
 
-`repr(packed)` forces Rust to strip any padding, and only align the type to a
-byte. This may improve the memory footprint, but will likely have other negative
+`repr(packed(n))` can lower the alignment of every single struct member and thus 
+the alignment of the struct itself. The final alignment of each struct member is 
+`min(n, normal_alignment)`. Hence, `packed`/`packed(1)` aligns each field to a 
+one-byte boundary. Effectively, this strips any padding between member fields, 
+which may improve the memory footprint, but will likely have other negative 
 side-effects.
 
-In particular, most architectures *strongly* prefer values to be aligned. This
-may mean the unaligned loads are penalized (x86), or even fault (some ARM
-chips). For simple cases like directly loading or storing a packed field, the
```

Comment:
I can remove this from the PR if you want. This was necessary so that each paragraph has roughly the same width in the markdown file.